CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v1.1.4](https://github.com/voxpupuli/container-puppetserver/tree/v1.1.4) (2024-01-30)
+
+[Full Changelog](https://github.com/voxpupuli/container-puppetserver/compare/v1.1.3...v1.1.4)
+
+**Implemented enhancements:**
+
+- build latest version for each puppet version [\#45](https://github.com/voxpupuli/container-puppetserver/pull/45) ([tuxmea](https://github.com/tuxmea))
```

Comment:
Since those are enhancements, it should be a minor release, not a patch release.

Reply:
hmm okay, was htinking about that too, but was not sure